packages/token-list/src/test/index.ts

```diff
@@ -64,7 +64,7 @@ describe( 'token-list', () => {
 
 		it( 'sets to stringified value', () => {
 			const list = new TokenList();
-			list.value = undefined;
+			list.value = undefined as unknown as string;
```

Comment:
For a case like this, I'd go with expect error. We know the type is wrong and we expect to have an error reported:

```suggestion
			// @ts-expect-error The value should be a string, for testing we pass a "bad" value.
			list.value = undefined;
```